llarp/handlers/exit.cpp

```diff
@@ -837,18 +809,14 @@ namespace llarp
     void
     ExitEndpoint::RemoveExit(const exit::Endpoint* ep)
     {
-      auto range = m_ActiveExits.equal_range(ep->PubKey());
-      auto itr = range.first;
-      while (itr != range.second)
+      for (auto [itr, end] = m_ActiveExits.equal_range(ep->PubKey()); itr != end; ++itr)
       {
         if (itr->second->GetCurrentPath() == ep->GetCurrentPath())
         {
-          itr = m_ActiveExits.erase(itr);
+          m_ActiveExits.erase(itr);
```

Comment:
this erase will change the thing we are iterating over if it rehashes

Reply:
Yes, but as the next line says right before it returns, "// now ep is gone af"